internal/aiptest/create/create_time.go

```diff
@@ -35,8 +35,13 @@ var createTime = suite.Test{
 		f.P(ident.AssertNilError, "(t, err)")
 		f.P(ident.AssertCheck, "(t, msg.CreateTime != nil)")
 		f.P(ident.AssertCheck, "(t, !msg.CreateTime.AsTime().IsZero())")
-		// Allow Created == Now due to flakyness of clock in podman and colima
-		f.P(ident.AssertCheck, "(t, !msg.CreateTime.AsTime().After(", ident.TimeNow, "()))")
+		f.P("if ", protogen.GoIdent{GoName: "GOOS", GoImportPath: "runtime"}, " == \"darwin\" {")
```

Comment:
I would flip the order of `GoName` and `GoImportPath` to get the same order as in an import statement, but that's a minor thing

Reply:
Good one, sure. Fixed!